README.md

```diff
@@ -119,9 +119,7 @@ saveCityState(cityStateRegex.match(cityStateRegex)[1], cityStateRegex.match(city
 **Good**:
 ```javascript
 const cityStateRegex = /^(.+)[,\\s]+(.+?)\s*(\d{5})?$/;
-const match = cityStateRegex.match(cityStateRegex)
-const city = match[1];
-const state = match[2];
+const [, city, state] = cityStateRegex.match(cityStateRegex);
```

Comment:
I find destructuring cleaner as well. For the record, that's my opinion and not Clean Code's. 

This is a great fix all in all! Can you fix the regex above so that we don't have to omit the first item in the array? I should have used a better one to begin with.